cpp/include/cuvs/neighbors/ivf_pq.h

```diff
@@ -262,23 +262,60 @@ cuvsError_t cuvsIvfPqIndexCreate(cuvsIvfPqIndex_t* index);
 cuvsError_t cuvsIvfPqIndexDestroy(cuvsIvfPqIndex_t index);
 
 /** Get the number of clusters/inverted lists */
-uint32_t cuvsIvfPqIndexGetNLists(cuvsIvfPqIndex_t index);
+cuvsError_t cuvsIvfPqIndexGetNLists(cuvsIvfPqIndex_t index, uint32_t* n_lists);
 
 /** Get the dimensionality */
-uint32_t cuvsIvfPqIndexGetDim(cuvsIvfPqIndex_t index);
+cuvsError_t cuvsIvfPqIndexGetDim(cuvsIvfPqIndex_t index, uint32_t* dim);
```

Comment:
https://github.com/rapidsai/cuvs/pull/1272 is using int64_t for these getters for ivf-flat and CAGRA? Can we do the same?